src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -32,4 +35,10 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
         return bookService.save(bookRequestDto);
     }
+    
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }
```

Comment:
You also need to add update method:
PUT: /api/books/{id} (Update a specific book)